src/controller/CHIPDevice.h

```diff
@@ -485,6 +493,9 @@ class Device : public Messaging::ExchangeDelegate, public SessionEstablishmentDe
      */
     Transport::PeerAddress mDeviceAddress = Transport::PeerAddress::UDP(Inet::IPAddress::Any);
 
+    uint32_t mDeviceMrpIdleInterval   = CHIP_CONFIG_MRP_DEFAULT_INITIAL_RETRY_INTERVAL;
```

Comment:
Also we have "Idle" and "Initial" which is confusing for me a little.

Reply:
Yes and that's basically because of that module: https://github.com/project-chip/connectedhomeip/blob/master/src/messaging/ReliableMessageProtocolConfig.h#L121  Currently MRP works in a non-spec compliant way, where INITIAL interval is used for the first retransmission instead of IDLE interval used in the idle mode. That needs to be refactored and changing the name from initial to idle is not quite true considering current state of implementation. Probably I shouldn't call other values "idle", but we are actually in the middle of step by step changes, so I didn't care about it that much. Do you think we should change all initial names to idle even if MRP engine will still treat it as initial interval?